api/api.proto

```diff
@@ -890,6 +890,26 @@ message CapacityUsageInfo {
   int64 total_bytes = 3;
 }
 
+message VolumeUsage {
+   // VolumeID is the id for the volume
+  string volume_id = 1;
+  // VolumeName is name of the volume
+  string volume_name = 2;
+  // PoolUUID is the UUID of the pool that this volume belongs to
+  string pool_uuid = 3;
+  // ExclusiveBytes size exclusively used by the volume
+  uint64 exclusive_bytes = 4;
+  // TotalBytes used by the volume
+  uint64 total_bytes = 5;
+  // LocalCloudSnapshot is true if this volume is snapshot created by cloudbackups
+  bool local_cloud_snapshot = 6;
```

Comment:
is this required ? can it be inferred from the volume name ?

Reply:
yes. It can be.